assets/ExpressionMetadata.json

```diff
@@ -639,6 +639,13 @@
             "minimumArguments": 1,
             "maximumArguments": 1
         },
+        {
```

Comment:
Thanks!

Please only add it to ExpressionMetadata.json, remove it from the test data (some of this original test structure was not written very well).  Sorry you had to find that out the hard way.

Reply:
@StephenWeatherford do you mean revert the changes to all three of the test files I modified?

Reply:
Yes.  Sorry for the confusion.  I've add comments to help others in the future (https://github.com/microsoft/vscode-azurearmtools/pull/1136).  TestData.ExpressionMetadata.json was meant to isolate you from having to update unit tests when changing ExpressionMetadata.json (except as opted in for functional tests).

Reply:
FYI, I'm back in the office now so will try to be quicker to respond from now on.